# Whoâ€™s that pathogen?!

```{=html}
<div style="font-family:monospace; font-size: 0.9em; background:#f0f0f0; border-left: 5px solid #4CAF50; padding: 0.75em 1em; margin-top: 1em; border-radius: 6px; width: fit-content;">
â¬œ Developing Hypotheses<br>
â¬œ Sample Collection<br>
â¬œ Outbreak Investigation<br>
ðŸŸ© <strong>Sequencing</strong><br>
â¬œ Bioinformatics<br>
â¬œ Molecular Epidemiology <br>
â¬œ Public Health Implementation
</div>


After much discussion, the team of Genomic Specialists decided that since the bat trainers were the earliest known cases to exhibit flu-like symptoms but test negative for flu, these samples would be ideal to start with. The team of Genomic Specialists discussed amongst themselves which kinds of sequencing approaches would be best to help identify the presence of microorganisms in the samples. Since the initial testing and sequencing that was performed on the muubats did not detect any pathogens known to cause any human illnesses, the team thought it would be best to start broad. The Bioinformaticians and Microbiologists decided to take a shotgun metagenomics approach and sequence everything in the samples collected from the bat trainers. 

Shotgun metagenomics sequencing allows microbiologists to sample all organisms in a mixed and complex sample. This sequencing and bioinformatics approach is comprehensive and allows us to evaluate microbial diversity, that is identify the types of species in a sample, as well as detect the relative abundance of certain microbes in the sample. Relative abundance is an inferred estimate of the percentage composition of a particular kind of organism relative to the totality of organisms detected and identified in a mixed sample. Oftentimes relative abundances measure on a scale from 0 to 1. 

The Microbiologists sequenced everything that could be present in the samples using shotgun metagenomics. Using this approach, nucleic acids, such as DNA and RNA is fragmented into smaller components which are then independently sequenced. Once the sequencing is complete the Bioinformaticians performed QC on the raw data to check for contaminants and background noise. Once everything looked good, the Bioinformaticians proceeded to process the raw sequencing data using their metagenomic pipelines. Their pipelines have been rigorously validated and was specially configured with tools to identify microbial species against the Internal Sequencing Database that is maintained by the Regional Health Lab, as well as a built-in algorithm that infers the relative abundance of microorganisms in the sample.

![](../images/relabundance_1.jpeg)

*Relative Abundance of Microorganisms in Swab Samples of Bat Trainers Collected 31-01-08.*
The relative abundance chart shows the percent composition in terms of the proportions of species in the swab samples taken from the Bat Trainers. The relative abundance is plotted on the vertical axis, with the Bat Trainers on the horizontal axis. Species were identified using the Internal Sequencing Database maintained by the Regional Health Lab.

## Discussion Six: Interpretation Metagenomics Sequencing Results and Visualizations

Looking at the chart showing the relative abundance of microorganisms in the swab samples collected from the bat trainers, how would you interpret the figure? For example, how many primary species were identified? Were there any species that were found in the sample that did not show any match to the Internal Sequencing Database? What do you think that might mean?


---

[View Timeline of Outbreak Events](../images/timeline_metagenomics.JPEG)