# Whoâ€™s that pathogen?!

After much discussion, the team of Genomic Specialists decided that since the bat trainers were the earliest known cases to exhibit flu-like symptoms but test negative for flu, these samples would be ideal to start with. The Bioinformaticians and Microbiologists decided to take a shotgun metagenomics approach and sequence everything in the samples collected from the bat trainers. Shotgun metagenomics sequencing allows microbiologists to sample all organisms in a mixed and complex sample to evaluate microbial diversity and detect the abundance of certain microbes in the sample. The Microbiologists sequenced everything that could be present in the samples, and the Bioinformaticians utilized their metagenomic pipelines to infer the relative abundance of microorganisms in the samples.

![](../images/relabundance_1.jpeg)

*Relative Abundance of Microorganisms in Swab Samples of Bat Trainers Collected 31-01-08.*
The relative abundance chart shows the proportions of species in the swab samples taken from the Bat Trainers. The relative abundance is plotted on the vertical axis, with the Bat Trainers on the horizontal axis. Species were identified using the Internal Sequencing Database maintained by the Regional Health Lab.

## Discussion Six:

Looking at the chart showing the relative abundance of microorganisms in the swab samples collected from the bat trainers, how would you interpret the figure? How many primary species were identified? Were there any species that were found in the sample that did not show any match to the Internal Sequencing Database?