# The Novel Pathogen

**Isolating and identifying the novel pathogen:**

In mid-May after the team identified an unknown microorganism in the sample, the Microbiologists began working with the specimens collected from the bat trainers to isolate the suspect pathogen. After several weeks of meticulous labor, the Microbiologists were successfully able to isolate the unknown pathogen from both samples. 
Using the raw sequencing data, the Bioinformaticians were able to utilize their pipelines to piece together a genome from the samples taken from one of the bat trainers. They ran the genome through the Public Genome Database to see if the sequences shared any homology or similarities with any known microorganisms. Fortunately, the results came back with a hit, demonstrating that the consensus genome belongs to a family of positive-sense RNA viruses called Pokeviridae. 

Additionally, the Bioinformaticians were able to identify a sequence in the Public Genome Database that appeared relatively similar to the genome theyâ€™ve assembled. Unfortunately, the metadata regarding this reference sequence is incomplete, as it is the only genome of its kind, and was assembled from genetic material taken from muutubats on June 14th of 2029. The muutubats are a species of fruit bats that shared a very recent common ancestor with the muubats and even live within the same geographic region. The Bioinformaticians decided to use the consensus genome of the virus taken from the muutubat as a reference genome.

The Bioinformaticians continued to work tirelessly to annotate the genome of the unknown virus, and fortunately were successful in mapping out most of the structural genes of the virus. It appeared that this virus contains 4 structural genes: a nucleocapsid, a spike protein, envelope, and matrix protein. However, much work was needed to be able to identify the non-structural proteins encoded within the four open reading frames.

![Consensus Genome of Novel Virus](../images/virus_genome.jpg)

*Annotated Genome of Novel Virus found in Muutubat.*
The figure shows the consensus genome pulled from the Public Genome Database and annotated using various gene annotation tools by the Bioinformatician. The consensus genome used is a high quality complete genome with 0% ambiguity in coverage across the genome. Genes were identified by homology to other viral genes in the Public Gene Database. Until the open reading frames are annotated, complete annotation of the genome remains incomplete.

Now that sequencing had confirmed the presence of a novel virus, the team turned to its genome for answers. The structure and composition of the viral genome could provide crucial clues about how it spreads, mutates, and interacts with its host. What could its genomic features reveal about its transmission and evolution?

## Discussion Seven

Given the information above, how would you interpret the genome? For example, what is the length of the genome? How many structural proteins does this genome appear to contain and what do the genes appear to code for? How many open reading frames do you notice? What are some examples of non-structural proteins that could be encoded within the open reading frames?