# Genomic Specialists

```{=html}
<div style="font-family:monospace; font-size: 0.9em; background:#f0f0f0; border-left: 5px solid #4CAF50; padding: 0.75em 1em; margin-top: 1em; border-radius: 6px; width: fit-content;">
â¬œ Developing Hypotheses<br>
â¬œ Sample Collection<br>
ðŸŸ© <strong>Outbreak Investigation</strong><br>
â¬œ Sequencing<br>
â¬œ Bioinformatics<br>
â¬œ Genomic Epidemiology <br>
â¬œ Public Health Implementation
</div>


**A team of Genomic Specialists were assigned to investigate the mystery flu-like outbreak in Fuchsia City/Safari Zone:**

Doctors were puzzled at the mystery flu-like illness that was spreading in Fuchsia City and Safari Zone. The doctors, specialists, researchers, and Local Health Epidemiologists worked together and decided to ask the Regional Health Department to investigate the outbreak to prevent the spread of the illness. A team of Genomics Specialists were assigned to investigate the city-wide and wildlife outbreak, including Microbiologists that specialized in laboratory methods and sequencing, Bioinformaticians that specialized in genome assembly and genomic analyses, and Molecular Epidemiologists that specialized in hypothesis testing, genomic analysis interpretation, and visualization. The Zoonotic Epidemiology Team and Local Health Epidemiologists were also assigned to work closely with the team to investigate the outbreak.

# All isolates were sent to the regional lab for isolation, testing, and sequencing:

The team of Genomic Specialists worked with the doctors and animal trainers to have the swab samples shipped over to the Regional Health Lab for further testing and sequencing. These include samples that were collected from Safari Zone workers, trainers, residents, felines, canines, ducklings, and even the two bat trainers. 

In terms of capacity, the lab has the capacity to perform short-read sequencing, long-read sequencing, and shotgun metagenomics on the samples. The Bioinformaticians had pipelines in place for genome assembly, metagenomic analyses, taxonomic identification, and annotation. Molecular Epidemiologists complemented this work by performing sequence alignments, pairwise distance analyses, and phylogenetic tree construction.


| Host      | Number of Samples |
|-----------|-------------------|
| Humans    | 23                |
| Felines   | 5                 |
| Canines   | 6                 |
| Ducklings | 4                 |


*Table showing number of swab samples collected, grouped by host species.*
There were 23 human samples, 5 feline samples, 6 canine samples, and 4 duck samples collected by the doctors and animal trainers and shipped over to the Regional Health Lab. Some swab samples collected in February yielded insufficient material for sequencing. As a result, only well-preserved samples from January through early February were sent to the Regional Health Lab for further analysis.


## Discussion Three:
The Regional Health Lab has the ability to culture bacteria and viruses for sequencing. In terms of sequencing methods and protocol, if we are trying to identify a novel bacteria and virus, what kind of sequencing platforms and protocols would you try to use in the initial identification? Be sure to fully explain your answer.


## Discussion Four:
Using your answer in Discussion Three, what type of bioinformatics methods or analyses would be useful in identifying a novel pathogen? 
